python/dgl/transform.py

```diff
@@ -987,6 +987,9 @@ def to_block(g, dst_nodes=None, include_dst_in_src=True):
             raise ValueError(
                 'Graph has more than one node type; please specify a dict for dst_nodes.')
         dst_nodes = {g.ntypes[0]: dst_nodes}
+    dst_nodes = {
+        ntype: utils.toindex(nodes, g._idtype_str).tousertensor()
```

Comment:
Why we need to do this?

Reply:
Sometimes the arguments may be a list and we need to convert it to a tensor.